packages/platform-browser/src/IndexedDbDatabase.ts

```diff
@@ -0,0 +1,191 @@
+/**
+ * @since 1.0.0
+ */
+import type * as Effect from "effect/Effect"
+import type * as Layer from "effect/Layer"
+import { type Pipeable } from "effect/Pipeable"
+import type * as IndexedDb from "./IndexedDb.js"
+import type * as IndexedDbQueryBuilder from "./IndexedDbQueryBuilder.js"
+import type * as IndexedDbTable from "./IndexedDbTable.js"
+import type * as IndexedDbVersion from "./IndexedDbVersion.js"
+import * as internal from "./internal/indexedDbDatabase.js"
+
+/**
+ * @since 1.0.0
+ * @category type ids
+ */
+export const TypeId: unique symbol = internal.TypeId
+
+/**
+ * @since 1.0.0
+ * @category type ids
+ */
+export type TypeId = typeof TypeId
+
+/**
+ * @since 1.0.0
+ * @category type ids
+ */
+export const ErrorTypeId: unique symbol = internal.ErrorTypeId
+
+/**
+ * @since 1.0.0
+ * @category type ids
+ */
+export type ErrorTypeId = typeof ErrorTypeId
+
+/**
+ * @since 1.0.0
+ * @category errors
+ */
+export const IndexedDbDatabaseError = internal.IndexedDbDatabaseError
```

Comment:
This needs an explicit type.

Reply:
Updated the error definition for internal/exported value and type